st2common/st2common/content/bootstrap.py

```diff
@@ -83,7 +83,7 @@ def register_opts():
 register_opts()
 
 
-def setup_virtualenvs():
+def setup_virtualenvs(recreate_virtualenvs=False):
```

Comment:
I think the tests might be failing because thee isn't a setup-recreate-virtualenvs in the cfg.BoolOpt like there is above for setup-virtualenvs.